src/List-item.js

```diff
@@ -13,13 +13,13 @@ class TodoItem extends Component {
 
         return (
 
-            <li>
+            <li className={todo.done ? 'done' : ''}>
 
                 <input type="checkbox"
-                    onChange={this.props.handleDone.bind(null, title)}
+                    onChange={this.props.handleDone.bind(null, todo.id)}
```

Comment:
We can also write this way
`onChange={() => this.props.handleDone(todo.id)}`

It's good to understand how `bind` works.